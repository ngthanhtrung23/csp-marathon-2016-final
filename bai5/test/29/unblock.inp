123334
12..54
120056
778.56
..8999
AAA...
