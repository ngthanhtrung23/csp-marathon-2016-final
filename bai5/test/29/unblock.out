27
A R 3
8 D 1
7 R 2
1 D 3
2 D 3
0 L 2
3 L 1
5 U 1
7 R 1
8 U 3
7 L 1
5 D 1
9 L 1
6 D 1
4 D 1
3 R 2
8 U 1
0 R 2
2 U 2
9 L 1
A L 2
5 D 2
0 R 1
8 D 1
3 L 1
4 U 1
0 R 1

